projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.html

```diff
@@ -2,9 +2,9 @@
 <ng-container #toolbarOutlet></ng-container>
 
 <div class="igx-grid__thead">
-    <div class="igx-grid__thead-wrapper" (keydown.meta.c)="copyHandler($event)" (keydown.control.c)="copyHandler($event)" (copy)="copyHandler($event)" tabindex="0" (focus)="navigation.focusFirstCell()" [attr.aria-activedescendant]="activeDescendant"
+    <div class="igx-grid__thead-wrapper" (keydown.meta.c)="copyHandler($event)" (keydown.control.c)="copyHandler($event)" (copy)="copyHandler($event)" tabindex="0" (focus)="navigation.focusFirstCell()"
      (keydown)="navigation.headerNavigation($event)" role="rowgroup" [style.width.px]='calcWidth' #theadRow>
-        <div class="igx-grid__tr" role="row" [style.width.px]='calcWidth'>
+        <div class="igx-grid__tr" role="row" [attr.aria-activedescendant]="activeDescendant" [style.width.px]='calcWidth'>
             <span *ngIf="hasMovableColumns && draggedColumn && pinnedColumns.length <= 0"
```

Comment:
@Aleksandyr , aria-activedescendant attribute should be attached to the element that holds the focus/ is marked with tabIndex > = 0/. You can see that if you enable some screen reader if you click on a cell and start navigation with the arrow keys it doesn't manage to read the cell content correctly.